docs/user-guide/configuration.md

```diff
@@ -137,7 +137,7 @@ Or starting with `stylelint-config-standard`, then extending layering `myExtenda
 
 **The value of `"extends"` is a "locater" (or an array of "locaters") that is ultimately `require()`d, so can fit whatever format works with Node's `require.resolve()` algorithm.** That means the a "locater" can be:
 
-- The name of a module in `node_modules` (e.g. `stylelint-config-standard`; that module's `main` file must be a valid JSON configuration)
+- The name of a module in `node*modules` (e.g. `stylelint-config-standard`; that module's `main` file must be a valid JSON configuration)
```

Comment:
Oops, I was a bit overly zealous with the find and replace :)

I'll fix it now.